bedrock/press/templates/press/press-inquiry.html

```diff
@@ -1,104 +1,87 @@
-{% extends "base-resp.html" %}
+{% extends "base-protocol-mozilla.html" %}
 
 {% block page_title %}{{_('Press Inquiry Form')}}{% endblock %}
-{% block body_class %}sand{% endblock %}
 
 {% block page_css %}
-  {{ css_bundle('press') }}
+{{ css_bundle('press') }}
 {% endblock %}
 
 {% block content %}
-
-<main role="main" id="main-content">
-  <header class="main-header">
-    <h1 class="title-shadow-box">{{_('Press Inquiry')}}</h1>
-  </header>
-
-  <section class="container">
-
+<main class="mzp-l-content">
+  <h1>{{_('Press Inquiry')}}</h1>
+  <section class="mzp-c-emphasis-box">
```

Comment:
Not a blocker, but visually it feels a little disjointed to put the whole form inside an emphasis box. The page margins don't line up, and there's not much space between the main headline and the form?

I don't think we need to try and maintain the design of these original pages, and using regular Protocol [article layout](https://protocol.mozilla.org/demos/article.html) is probably OK? There's a [base template](https://github.com/mozilla/bedrock/blob/master/bedrock/base/templates/base-article.html) to help doing this.

![image](https://user-images.githubusercontent.com/400117/68658487-71416600-052d-11ea-84d6-ac61201da68d.png)

Reply:
Thanks Alex, this is super helpful. Totally agree & will update. :+1: